### DISCUSSION TITLE
How to Add Entry?

### DISCUSSION THREAD

**Discussion by maroonedsia** [Kunzisoft/KeePassDX#1896] [Q&A] [answered]
Hi, I don't see any options for adding a new entry! Am I missing anything, or it's not possible?! Thanks! 👍

**Comment by maroonedsia** [comment]
p.s. On Google Play page, the latest KeePassDX is 4.4, but on my device it is 4.0.8 and Google Play does NOT show the update button for me.  So,
A. Is this a known issue?
B. Is this related to the bug above? (On the screenshots I see the + button to add entries, but I don't see that on my device!)

**Reply by shuvashish76** [reply]
The latest version on Google Play is 4.0.8. Make sure to to download the official app : https://play.google.com/store/apps/details?id=com.kunzisoft.keepass.free

**Comment by J-Jamet** [comment, marked as answer]
Check that you are not in Read Only mode, which you can change by clicking on the pen button on the main credentials screen.
_Note that some file managers force a file to be opened read-only, in which case you need to contact the app providing the file._
